### ISSUE TITLE
Integrating a ci pipeline into the project

### ISSUE THREAD

**Issue by alexprodan99** [fireship-io/flamethrower#27]
**What about integrating a ci pipeline into the project?**
In this way, we could automate the process of checking that the new changes are breaking the codebase or not.
I think that a Github Actions pipeline would fit well here.

**Comment by ekwoka** [comment]
@b0iq Are you running the example site?

It's quite annoying currently, for me it was 3 terminals, one running the liveserver, one watching to build the package, and one that was running the tests when the package changed 😤

**Comment by b0iq** [comment]
@ekwoka 

Yeah, me too, I'm running three terminals just like you and I did install playwright but it keeps failing 🤬

**Comment by dhirajgagrai** [comment]
@b0iq Which of the tests are failing for you?

**Comment by micziz** [comment]
Test cloud be easily automated. Even running the example site (even tough I think leerobs pr to add vercel should have done it?) Anyways:  👍
